integration-tests/runtime-tests/test/tests/crowdloanRewards/testHandlers/crowdloanHandler.ts

```diff
@@ -0,0 +1,188 @@
+import {KeyringPair} from "@polkadot/keyring/types";
+import {sendAndWaitForSuccess, sendUnsignedAndWaitForSuccess} from "@composable/utils/polkadotjs";
+import {IKeyringPair} from "@polkadot/types/types";
+import {PalletCrowdloanRewardsModelsRemoteAccount} from "@composable/types/interfaces";
+import {u128, u32} from "@polkadot/types-codec";
+import {shares} from "@composabletests/tests/crowdloanRewards/contributions.json";
+import {expect} from "chai";
+
+
+const toHexString = bytes =>
+  Array.prototype.map.call(bytes, x => ('0' + (x & 0xFF).toString(16)).slice(-2)).join('');
```

Comment:
hmm... there is [toHex()](https://polkadot.js.org/docs/api/start/types.basics#everything-is-a-type).
it doesn't suit?

Reply:
Actually don't know. This was already in the original implementation by Hussein.
I noted it down to check it for the next update.

Thanks for your input. :)